en/middleware/xrce_dds.md

```diff
@@ -241,23 +244,97 @@ The topics are release specific (support is compiled into [microdds-client](../m
+- If you're creating your own new topics e/o uORB messages you will need to update [dds_topics.yaml](https://github.com/PX4/PX4-Autopilot/blob/main/src/modules/microdds_client/dds_topics.yaml), clone the the interface package and then manually synchronize it by copying the new/modified message definitions from [PX4-Autopilot/msg](https://github.com/PX4/PX4-Autopilot/tree/main/msg) to its `msg` folders. Assuming that PX4-Autopilot is in your home directory `~` while `px4_msgs` is in `~/px4_ros_com/src/`, then the command can be
+  ```sh
+  rm ~/px4_ros_com/src/px4_msgs/msg/*.msg
+  cp ~/PX4-Autopilot/mgs/*.msg ~/px4_ros_com/src/px4_msgs/msg/
+  ```
   
   :::note
-  Technically the messages must be in a folder defined by the `type` prefix in the yaml file:
-  As you can see below, this is `px4_msgs::msg::`:
-  
-  ```yaml
-  - topic: /fmu/out/vehicle_odometry
-    type: px4_msgs::msg::VehicleOdometry
+  Technically, [dds_topics.yaml](https://github.com/PX4/PX4-Autopilot/blob/main/src/modules/microdds_client/dds_topics.yaml) completely defines the pairing between PX4 message and topics and the ROS 2 ones.
+  It is structured as follows:
+  ```
+  publications:
+
+    - topic: /fmu/out/collision_constraints
+      type: px4_msgs::msg::CollisionConstraints
+
+    ...  
+
+    - topic: /fmu/out/vehicle_odometry
+      type: px4_msgs::msg::VehicleOdometry
+
+    - topic: /fmu/out/vehicle_status
+      type: px4_msgs::msg::VehicleStatus
+
+    - topic: /fmu/out/vehicle_trajectory_waypoint_desired
+      type: px4_msgs::msg::VehicleTrajectoryWaypoint
+
+  subscriptions:
+
+    - topic: /fmu/in/offboard_control_mode
+      type: px4_msgs::msg::OffboardControlMode
+
+    ...
+
+    - topic: /fmu/in/vehicle_trajectory_waypoint
+      type: px4_msgs::msg::VehicleTrajectoryWaypoint
   ```
+  
+  Each (`topic`,`type`) pairs defines
+
+  1. A new subscription or publication depending on the list to which it is added.
+  2. The topic _base name_, which **must** coincide with the desired uORB topic name that you want to publish/subscribe.
+     It is identified by the last token in `topic:` that starts with `/` and does not contains any `/` in it.
+     `vehicle_odometry`, `vehicle_status` and `offboard_control_mode` are examples of base names.
+  3. The topic [namespace](https://design.ros2.org/articles/topic_and_service_names.html#namespaces). 
+     By default it is set to:
+     
+      - `/fmu/out/` for topics that are _published_ by PX4.
+      - `/fmu/in/` for topics that are _subscribed_ by PX4.
+  4. The message type (`VehicleOdometry`, `VehicleStatus`, `OffboardControlMode`, etc.) and the ROS 2 package (`px4_msgs`) that is expected to provide the message definition.
+
+  You can arbitrarily change the configuration.
+  For example, you could use different default namespaces or use a custom package to store the message definitions. 
   :::
 
 
+### Customizing the topic namespace
```

Comment:
This is a lower level heading (3) than the parent (2 - ""Supported uORB Messages"). Is that the intent? If not, perhaps this should go up one level.